cmd/login/clientcert.go

```diff
@@ -446,11 +542,11 @@ func getCertOperatorVersion(ctx context.Context, releaseService release.Interfac
 }
 
 func validateProvider(provider string) error {
-	if provider == key.ProviderAWS || provider == key.ProviderAzure {
-		return nil
+	if provider != key.ProviderAWS && provider != key.ProviderAzure && provider != key.ProviderOpenStack {
```

Comment:
Maybe it makes sense to check for providers that aren't supported instead of those that are--I think it would just be KVM.

Reply:
Yeah I did it that way at first.
However, new providers are now being added more frequently (like google the other day) and it might be better to consciously enable them IMO

Reply:
All new providers will be pure CAPI so I feel the opposite--it's an extra chore to manually allow them versus allowing them by default.

Reply:
Yeah, this makes sense if we replace the checks for openstack with a check for "pure capi" as you suggested below. :)